src/clj/game/core/moving.clj

```diff
@@ -518,6 +514,24 @@
         (when (in-hand? moved-b) (add-to-currently-drawing state b-side moved-b)))
       [(get-card state moved-a) (get-card state moved-b)])))
 
+(defn swap-cards-async
+  "Swaps two cards when one or both aren't installed"
+  ([state side a b] (swap-cards-async state side nil a b))
```

Comment:
If a function is async, it shouldn't be possible to call it incorrectly. Just remove this arity please.